src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -0,0 +1,50 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module provides the JumpStart Curated Hub class."""
+from __future__ import absolute_import
+
+from typing import Optional, Dict, Any
+from sagemaker.jumpstart.constants import DEFAULT_JUMPSTART_SAGEMAKER_SESSION
+
+from sagemaker.session import Session
+
+
+class CuratedHub:
```

Comment:
why not call this class `Hub`? It can be more than just a `CuratedHub`

Reply:
That's what was agreed upon with SDK team. Ideally, we would have a base `Hub` class that `CuratedHub` extends.

My main reasoning for calling it this is that it'll have `CuratedHub`-specific functions like `sync`. It won't try and solve any `Hub` related tasks